download.html

```diff
@@ -61,6 +57,11 @@ <h2>Linux Installation</h2>
 
 <div>
   <h2>Additional Links</h2>
+
+  <p><a id="win32" href="https://github.com/triplea-game/triplea/releases/latest" target="_blank"
+     onclick="hideShowInstallationText(document.getElementById('windows-install'))">
+    Download TripleA&nbsp;for Windows <strong>32&nbsp;bit</strong>
+  </a></p>
   <p><a href="{{ '/release_notes/' | prepend: site.baseurl }}">Release Notes</a></p>
   <p><a href="https://github.com/triplea-game/triplea/wiki/Verifying-Binaries">Verifying Binaries</a></p>
   <p><a href="{{ '/old_downloads/' | prepend: site.baseurl }}">Previous TripleA Releases</a></p>
```

Comment:
A couple of lines below there's some javascript if else block that does show the DL size for the hovered download entry, explicitly stating "win32", that could be removed.

Reply:
I'm reluctant to remove too much just yet; in case we see a sizable number of 32 bit downloads ongoing after this update, then we'll want to restore to the old image. Merge conflicts could come up if we just rely on revert only. I'm personally okay keeping some tech debt here. Let me know if you really think we should do the cleanup right now.